orchestrator-salt/src/main/java/com/sequenceiq/cloudbreak/orchestrator/salt/SaltOrchestrator.java

```diff
@@ -1315,15 +1318,15 @@ public Map<String, String> getMembers(GatewayConfig gatewayConfig, List<String>
     }
 
     @Override
-    public void backupDatabase(GatewayConfig primaryGateway, Set<String> target, SaltConfig saltConfig, ExitCriteriaModel exitModel)
+    public void backupDatabase(GatewayConfig primaryGateway, Set<String> target, SaltConfig saltConfig, ExitCriteriaModel exitModel, Boolean isLongTimeBackupEnabled)
             throws CloudbreakOrchestratorFailedException {
-        callBackupRestore(primaryGateway, target, saltConfig, exitModel, DATABASE_BACKUP);
+        callBackupRestore(primaryGateway, target, saltConfig, exitModel, DATABASE_BACKUP, isLongTimeBackupEnabled);
     }
 
     @Override
     public void restoreDatabase(GatewayConfig primaryGateway, Set<String> target, SaltConfig saltConfig, ExitCriteriaModel exitModel)
```

Comment:
if we have long backup I expect restore to be long as well

Reply:
Added restore as well. Thank you!!